Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aws-nitro-enclaves-cose"
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
Ah, forgot. I will update this when I'm ready to publish a new revision. This also requires adding some extra information in the Changelog.

Reply:
Ok I reverted the version back to `0.4.0`.